test/tracker-component.js

```diff
@@ -0,0 +1,148 @@
+/* globals describe before it */
```

Comment:
I'd add a test in here somewhere that asserts that you can instantiate trackers with different users